### ISSUE TITLE
Improve performance of RNNS 

### ISSUE THREAD

**Issue by frodre** [ai2cm/fv3net#1564]
as measured by wall time to specific training loss level. Increase batch size, gpu, and learning rate.

**Comment by frodre** [comment]
[Report](https://wandb.ai/ai2cm/microphysics-emulation/reports/RNN-Runtime-Optimization-Checks--VmlldzoxMzYyNDkx)

[Current Branch w/ GPU adjustments](https://github.com/ai2cm/fv3net/tree/rnn-optimization)

GPU image: `us.gcr.io/vcm-ml/prognostic_run:cuda-base-v2`
